src/include/homestore/btree/btree.ipp

```diff
@@ -327,11 +327,12 @@ void Btree< K, V >::print_tree(const std::string& file) const {
     to_string(m_root_node_info.bnode_id(), buf);
     m_btree_lock.unlock_shared();
 
-    BT_LOG(INFO, "Pre order traversal of tree:\n<{}>", buf);
```

Comment:
we can log the btree to 2 places stdout/BT_LOG and file. file is mostly just for testing.